compiler/compile/OMRCompilation.cpp

```diff
@@ -978,6 +982,13 @@ int32_t OMR::Compilation::compile()
       methodInfo->setProfileInfo(NULL);
 #endif
 
+#if defined(LINUX)
+   // Create the compile time profiler
+   TR::CompileTimeProfiler *perf;
```

Comment:
This needs to be set NULL for the if clean up to work properly in call cases since perf is not guaranteed to be 0.